### ISSUE TITLE
iPhone 4S: Failed to load manifest error

### ISSUE THREAD

**Issue by MaximeLuk** [airsquared/blobsaver#592]
**Describe the bug**
When I try to save a blob for iPhone 4s, it gives me an error message "Failed to load manifest"

**Screenshots**

**Desktop:**
 - OS Version: iOS 9.3.6
 - iOS Device: iPhone 4S

**Checklist (place an x between the brackets to mark as completed):**
 - [x] I have made sure I am on the latest version of blobsaver
 - [x] I have checked other open issues to see if my problem has already been reported
 - [x] I have tried common troubleshooting tactics such as restarting my computer and reinstalling blobsaver

**Log(s)**
```
iosVersions = [IOSVersion[versionString=9.3.6, buildid=13G37, ipswURL=http://updates-http.cdn-apple.com/2019/ios/041-80043-20190722-6C65AD27-69D8-499C-BC15-DE7AC74DE2BD/iPhone4,1_9.3.6_13G37_Restore.ipsw, signed=true], IOSVersion[versionString=9.3.5, buildid=13G36, ipswURL=http://appldnld.apple.com/iOS9.3.5/031-73068-20160825-6A2B99DE-6711-11E6-BB8F-133834D2D062/iPhone4,1_9.3.5_13G36_Restore.ipsw, signed=true]]
Running: [/Applications/blobsaver.app/Contents/MacOS/tsschecker, --nocache, --save, --device, iPhone4,1, --ecid, 1, --boardconfig, n94ap, --generator, 0x1111111111111111, --save-path, /Users/Shared/Blobs, --build-manifest, /private/var/folders/2v/pr7ry4913g97dbphmky5jmlm0000gn/T/BuildManifest10249483171916877001.plist]
tsschecker version: 0.419-ec6e6fdfe06c8b7a5f48983eb78738fa9b6a876d
libfragmentzip version: 0.1-aaf6fae83a0aa6f7aae1c94721857076d04a14e8-RELEASE
[TSSC] manually specified generator "0x1111111111111111"
[TSSC] manually specified ECID to use, parsed "1" to dec:1 hex:1
[Error] [TSSC] failed to load manifest

[Error] [TSSC] checking tss status failed!
```

**Comment by Merculous** [comment]
Getting same issue on Windows 10 with v3.5.1.

**Comment by airsquared** [comment]
I'll look into this. Why do you need to save blobs for such an old device in the first place?

**Comment by Merculous** [comment]
You'll never know when Apple will accidentally sign old iOS again. The last one, I happened to see it happen exactly when it started.

No hurt in saving blobs for absolutely everything that could randomly be signed again. Merely a just-in-case scenario, but useful for at least downgrading to 6.1.3 or 8.4.1 since their OTA's are still signed, so that's at least one purpose to still save blobs for 4S.

**Comment by MaximeLuk** [comment]
I want to downgrade to ios 6.1.3

**Comment by MaximeLuk** [comment]
I already went down to 8.4.1

**Comment by airsquared** [comment]
This has been fixed and will be released in the next version.

You can also use the latest [nightly build](https://nightly.link/airsquared/blobsaver/workflows/main/master) right now if you don't want to wait.

**Comment by KaanSystemsProjects** [comment]
I have a iPhone 4s with iOS 8.0 and have the same problem

**Comment by bonattomatheus** [comment]
> This has been fixed and will be released in the next version.
> 
> You can also use the latest [nightly build](https://nightly.link/airsquared/blobsaver/workflows/main/master) right now if you don't want to wait.

Now the issue is it cant extract "build manifest"

Log:

### **_airsquared.blobsaver.app.TSS$TSSException: Unable to extract BuildManifest.
	at airsquared.blobsaver@3.5.1/airsquared.blobsaver.app.TSS.saveFor(Unknown Source)
	at airsquared.blobsaver@3.5.1/airsquared.blobsaver.app.TSS.call(Unknown Source)
	at airsquared.blobsaver@3.5.1/airsquared.blobsaver.app.TSS.call(Unknown Source)
	at javafx.graphics@20/javafx.concurrent.Task$TaskCallable.call(Unknown Source)
	at java.base/java.util.concurrent.FutureTask.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.base/java.lang.Thread.run(Unknown Source)
Caused by: java.io.IOException: Error on ZipFile ipsw
	at blobsaver.merged.module@3.5.1/org.apache.commons.compress.archivers.zip.ZipFile.<init>(Unknown Source)
	at blobsaver.merged.module@3.5.1/org.apache.commons.compress.archivers.zip.ZipFile.<init>(Unknown Source)
	at airsquared.blobsaver@3.5.1/airsquared.blobsaver.app.Utils.extractManifestFromZip(Unknown Source)
	at airsquared.blobsaver@3.5.1/airsquared.blobsaver.app.Utils.extractBuildManifest(Unknown Source)
	... 8 more
Caused by: java.io.IOException: HTTP Response was 206 Partial Content
	at airsquared.blobsaver@3.5.1/airsquared.blobsaver.app.Network$HttpChannel.ensureOpen(Unknown Source)
	at airsquared.blobsaver@3.5.1/airsquared.blobsaver.app.Network$HttpChannel.read(Unknown Source)
	at blobsaver.merged.module@3.5.1/org.apache.commons.compress.utils.IOUtils.readFully(Unknown Source)
	at blobsaver.merged.module@3.5.1/org.apache.commons.compress.archivers.zip.ZipFile.tryToLocateSignature(Unknown Source)
	at blobsaver.merged.module@3.5.1/org.apache.commons.compress.archivers.zip.ZipFile.positionAtEndOfCentralDirectoryRecord(Unknown Source)
	at blobsaver.merged.module@3.5.1/org.apache.commons.compress.archivers.zip.ZipFile.positionAtCentralDirectory(Unknown Source)
	at blobsaver.merged.module@3.5.1/org.apache.commons.compress.archivers.zip.ZipFile.populateFromCentralDirectory(Unknown Source)
	... 12 more_**

This is on the new nightly version.

**Comment by airsquared** [comment]
Try again with the latest nightly version, it should fix it now.

**Comment by airsquared** [comment]
Fixed in v3.5.2.
